presto-docs/src/main/sphinx/release/release-0.155.rst

```diff
@@ -0,0 +1,23 @@
+=============
+Release 0.155
+=============
+
+General Changes
+---------------
+
+* Fixes incorrect results when queries contain multiple grouping sets that
+  resolve to the same set when unaliased
```

Comment:
We can remove the word unaliased completely - it can just be `resolve to the same set`